### ISSUE TITLE
OnComplete option is not available since v2

### ISSUE THREAD

**Issue by rapidfixer** [Stanko/animated-scroll-to#45]
Was it intended removal?

**Comment by Stanko** [comment]
Hello @rapidfixer,
Indeed it was. Now the method returns a promise, so you can just use `.then()`, like this:

```js
animateScrollTo(500).then(hasScrolledToPosition => {
  // scroll animation is finished

  // "hasScrolledToPosition" indicates if page/element
  // was scrolled to a desired position
  // or if animation got interrupted
  if (hasScrolledToPosition) {
    // page is scrolled to a desired position
  } else {
    // scroll animation was interrupted by user
    // or by another call of "animateScrollTo"
  }
});
```

Hope that helps.
Cheers!

**Comment by rapidfixer** [comment]
Thanks, that was not obvious looking at _d.ts_ because of returning **any** type.

**Comment by Stanko** [comment]
That is true, let me try to fix that!

**Comment by Stanko** [comment]
Fixed in `2.0.9` 🎉
